pkgs/os-specific/darwin/m-cli/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "m-cli";
-  version = "0.2.5";
+  version = "0.3.0";
 
   src = fetchFromGitHub {
     owner = "rgcr";
-    repo = "m-cli";
+    repo = pname;
```

Comment:
FYI I am not a fan of this because when pname gets normalized in python this can silently mess things up.

Reply:
Noted.